### ISSUE TITLE
The Adverbial "Kaf" (כ)

### ISSUE THREAD

**Issue by NathanD38** [IAHLT/UD_Hebrew#8]
@amir-zeldes

We wanted to ask regarding the `advmod` deprel origin of adverbial "Kaf" (כ) with the meaning _about_.
In the following sentence, should we connect the `advmod` from the numeral or from the noun?

 כתב היד המקורי התגלה כחמישים שנה לאחר תחילת החפירות באתר..

Should it be `nummod`(shana, xamishim) and `advmod`(shana, ke)?
Or `advmod`(xamishim, ke)?

**Comment by amir-zeldes** [comment]
This has actually been discussed in UD before, it's:

`advmod(xamishim, ke)`

**Comment by Hilla-Merhav** [comment]
@amir-zeldes Thanks! And for other adverbial modifiers of numerals – לפחות, בלבד:
רוב של 61 חברי כנסת **לפחות**
פורום של שני אנשים **בלבד**
Do we connect the לפחות and בלבד to the `NUM`, and not the `NOUN`, as we do with ke (פורום של כשני אנשים)?

**Comment by amir-zeldes** [comment]
The case of לפחות seems more clearly attached to the number:

כמה חברי כנסת?
לפחות 61

For שני אנשים, there is some ambiguity - even if we accept שני as a variant of שניים, it's a bit odd to say:

פורום של כמה אנשים?
שניים בלבד

And even if we accept it, it's still ambiguous:

just (two people)
(just two) people

But if we can't discern a specific meaning between these, the preference in existing UD treebanks is to attach it to the noun as a phrase adverbial:

http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=613fa78f9779a

**Comment by Hilla-Merhav** [comment]
@amir-zeldes Thanks! I want to make sure I understand - the right anaysis is: `advmod` (61, lefaxot), `advmod` (anashim, bilvad)? We treat lefaxot differently? or we should connect both of them to the NOUN?

**Comment by amir-zeldes** [comment]
Yes, lefaxot is a quantity modifier which attaches to numbers regularly. "bilvad" is more ambiguous, since you can have "(only 2) people", or "(only (2 people)" (similar to "only these two people", where it's clearly not a modifier of "2")

If we know for sure that "only" modifies a number, we can attach it that way, but if it's not totally clear, other TBs have treated "only" as an NP level modifier.
